routes/api/v1/meals.js

```diff
@@ -7,22 +7,61 @@ const fetch = require('node-fetch');
 router.get("/:id/foods", async function(req, res, next) {
   res.setHeader("Content-Type", "application/json");
   Meal.findOne({
-    where: {
-      id: req.params.id
-    },
-    attributes: ['id', 'name'],
-    include: [{model:Food, attributes: ['id', 'name', 'calories'], through: { attributes: []}}]
-  })
-  .then(meal => {
-    if(!meal){
-      res.status(404).send({ error: "Requested meal could not be found." });
-    }else{
-      res.status(200).send(JSON.stringify(meal));
-    }
-  })
-  .catch(error => {
-    res.status(404).send({error})
-  });
+      where: {
+        id: req.params.id
+      },
+      attributes: ['id', 'name'],
+      include: [{
+        model: Food,
+        attributes: ['id', 'name', 'calories'],
+        through: {
+          attributes: []
+        }
+      }]
+    })
+    .then(meal => {
+      if (!meal) {
+        res.status(404).send({
+          error: "Requested meal could not be found."
+        });
+      } else {
+        res.status(200).send(JSON.stringify(meal));
+      }
+    })
+    .catch(error => {
+      res.status(404).send({
+        error
+      })
+    });
 });
 
+router.get("/", async function(req, res, next) {
+  res.setHeader("content-Type", "application/json");
+  Meal.findAll({
+      attributes: ['id', 'name'],
+      include: [{
+        model: Food,
+        attributes: ['id', 'name', 'calories'],
+        through: {
+          attributes: []
+        }
+      }]
+    })
+    .then(meals => {
+      if (!meals) {
+        res.status(404).send({
+          error: 'There are no meals in the database.'
```

Comment:
Have you tested this? I believe no meals would result in an empty array and wouldn't hit here (you're not using a 'where' in the findAll as opposed to we did for the findOne). Don't think you need the if statement.

Reply:
I'll hop on testing this.